Doc/howto/functional.rst

```diff
@@ -1131,20 +1116,21 @@ usual way::
     def print_assign(name, value):
         return name + '=' + str(value)
 
-Which alternative is preferable?  That's a style question; my usual course is to
-avoid using ``lambda``.
+Which alternative is preferable?  That's mostly a question of style.
 
-One reason for my preference is that ``lambda`` is quite limited in the
+You may wish to avoid using ``lambda``, as there are limits to the
 functions it can define.  The result has to be computable as a single
 expression, which means you can't have multiway ``if... elif... else``
 comparisons or ``try... except`` statements.  If you try to do too much in a
 ``lambda`` statement, you'll end up with an overly complicated expression that's
-hard to read.  Quick, what's the following code doing? ::
+hard to read.
+
+Consider::
```

Comment:
The use of a rhetorical question aimed at the reader has the effect of shaking up the pace of the prose by suddenly throwing in a different kind of sentence construction. I think it's an enjoyable effect, and don't think it needs to be removed

Reply:
Yeah, most of the changes above to remove 'I suggest' or 'I think' are fine with me, but here the sentence is trying to ask a somewhat punchy question.